src/webapp/src/client/app/common/directives/wall/posts/post-create/post-create.component.js

```diff
@@ -133,5 +173,18 @@
             }
             $scope.$apply();
         }
+
+        function getUsersForAutocomplete(query) {
+            return kudosFactory.getUsersForAutoComplete(query);
```

Comment:
This seems wrong and breaks single-responsibility (Kudos are not related to Posts).
Recommendation is to either move autocomplete logic to some reusable component, or more simple solution - replicate same autocomplete call inside Posts-related module. (It would not be code duplication as per se, and it also shouldn't be a bit bunch of logic involved.)